drivers/SmartThings/zigbee-switch/src/init.lua

```diff
@@ -19,7 +19,23 @@ local clusters = require "st.zigbee.zcl.clusters"
 local configurationMap = require "configurations"
 local SimpleMetering = clusters.SimpleMetering
 local ElectricalMeasurement = clusters.ElectricalMeasurement
+local ColorControl = clusters.ColorControl
 local preferences = require "preferences"
+local utils = require "st.utils"
+
+local SANITY_CHECK_MIN_KELVIN = 1000
+local SANITY_CHECK_MAX_KELVIN = 10000
```

Comment:
Want to confirm that this `1,000K - 10,000K` range are the only values that we want for ColorTemperature. The spec states  a range from `15.32K - 1,000,000K`:

> _**ZCL Rev. 8 - 5.2.2.2.1.8 ColorTemperatureMireds Attribute**_
Color temperature in kelvins = 1,000,000 / ColorTemperatureMireds, where ColorTemperatureMireds is in
the range 1 to 65279 mireds inclusive, giving a color temperature range from 1,000,000 kelvins to 15.32
kelvins.

Reply:
Our capability only supports from 1 to 30,000 so I'll probably go with that.

In reality, I don't think we've seen any bulbs outside of this current range.